### DISCUSSION TITLE
product_number

### DISCUSSION THREAD

**Discussion by ghena** [bagisto/bagisto#6742] [Q&A]
Hi guys,
just a question.

in product module is define product_number field.

is possibile to delete or is necessary edit all repositories ?

Using a new productType is possible to remove it ?
Regards.
g.

**Comment by Amitk-webkul** [comment]
Hi @ghena,

when you delete or modify the product_number in the product table, it's essential to update the corresponding fields in the section of your form to prevent potential SQL syntax errors. Therefore, ensure that all your keys align with the column names to maintain proper functionality.
